readme_renderer/__main__.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, print_function
+import argparse
+from readme_renderer.rst import render
+import sys
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description="Renders a .rst README to HTML",
+    )
+    parser.add_argument('input', help="Input README file")
+    parser.add_argument('-o', '--output', help="Output file (default: stdout)")
+    args = parser.parse_args()
+    if args.output:
+        output_file = open(args.output, 'w')
+    else:
+        output_file = sys.stdout
+    input_file = open(args.input)
```

Comment:
You can use [`argparse.FileType`](https://docs.python.org/3/library/argparse.html#filetype-objects) here to eliminate the need for these lines.

Reply:
Specifically, the argument lines can become:

```python
parser.add_argument('input', help="Input README file", type=argparse.FileType('r'))
parser.add_argument('-o', '--output', type=argparse.FileType('w'), default='-', help="Output file (default: stdout)")
```

And then you can just do:

```python
rendered = render(args.input.read(), stream=sys.stderr)
if rendered is None:
    sys.exit(1)
print(rendered, file=args.output)
```

Reply:
Oh wow, I didn't know about `type=`. Neat!